src/base/emu-i386/simx86/codegen-sim.h

```diff
@@ -85,6 +85,7 @@ extern void FlagSync_All (void);
 extern void Gen_sim(int op, int mode, ...);
 extern void AddrGen_sim(int op, int mode, ...);
 extern void InitGen_sim(void);
+unsigned int CloseAndExec_sim(unsigned int PC, int mode);
```

Comment:
Don't you need the separate
Close() and Exec() now, similar
to jit? Otherwise you can't
integrate prejit, it needs Close()
w/o Exec().

I don't mean you need to
change this patch: maybe
you need it in another patch?